pkg/collector/corechecks/cluster/kubernetesapiserver/kubernetes_eventbundle.go

```diff
@@ -117,6 +113,15 @@ func (b *kubernetesEventBundle) formatEvents(clusterName string, providerIDCache
 		}
 	}
 
+	if b.namespace != "" {
+		tags = append(tags,
+			// TODO remove the deprecated namespace tag, we should
+			// only rely on kube_namespace
+			fmt.Sprintf("namespace:%s", b.namespace),
```

Comment:
Since it is used for new metrics, maybe only add the "official" tag which is `kube_namespace`

Reply:
forget about it, I just saw that we already set both tags before

Reply:
Oh this is not new, I just moved some code around: https://github.com/DataDog/datadog-agent/pull/12397/files#diff-ec73ad97ed2da8bc8d5fe9080ecde5e4aa2c87ead5b8a5fb87e36d38c6e68aebL132-L136